### PR TITLE
Add error classes to all the necesary input fields

### PR DESCRIPTION
**Pull request by jaysonhunter**

### CODE REVIEW

File: views/booking.njk [hunk 1]
```diff
@@ -48,10 +48,11 @@
     <h2 class="lbh-heading-h2">Booking details</h2>
 
     <h3 class="lbh-heading-h3">Hirer details</h3>
+govuk-input--error
```
Comment (LBHELewis) [inline views/booking.njk:51]: Why is this here?

File: views/booking.njk [hunk 2]
```diff
@@ -113,15 +114,15 @@
...
                 name="booking_email" 
                 type="text" 
                 value="{{ oldInput.email }}">
         </div>
 
         <div class="govuk-form-group lbh-form-group">
-            <label class="govuk-label  lbh-label" for="input-with-hint-text">
+            <label class="govuk-label lbh-label" for="input-with-hint-text">
                 Contact number (optional)
             </label>
```
Comment (LBHELewis) [inline views/booking.njk:128]: Should the contact number input be of type "tel"?

Comment (jaysonhunter) [reply views/booking.njk:128]: Good point. I'll make the change thanks.
